update vector search docs (#18779)
pingcap/docs: vector-search-get-started-using-sql.md

---
title: Get Started with Vector Search via SQL
summary: Learn how to quickly get started with Vector Search in TiDB using SQL statements to power your generative AI applications.
---

# Get Started with Vector Search via SQL

TiDB extends MySQL syntax to support [Vector Search](/vector-search-overview.md) and introduce new [Vector data types](/vector-search-data-types.md) and several [vector functions](/vector-search-functions-and-operators.md).

This tutorial demonstrates how to get started with TiDB Vector Search just using SQL statements. You will learn how to use the [MySQL command-line client](https://dev.mysql.com/doc/refman/8.4/en/mysql.html) to complete the following operations:

- Connect to your TiDB cluster.
- Create a vector table.
- Store vector embeddings.
- Perform vector search queries.

<CustomContent platform="tidb">

> **Warning:**
>
> The vector search feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

</CustomContent>

> **Note:**
>

> > The vector search feature is only available for TiDB Self-Managed clusters and [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters.

Reviewer: ```suggestion
> The vector search feature is only available for TiDB Self-Managed clusters and [TiDB Cloud Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-cloud-serverless) clusters.
```

</CustomContent>

## Get started

### Step 1. Connect to the TiDB cluster

Connect to your TiDB cluster depending on the TiDB deployment option you've selected.

<SimpleTab>
<div label="TiDB Cloud Serverless">

1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.

2. Click **Connect** in the upper-right corner. A connection dialog is displayed.

3. In the connection dialog, select **MySQL CLI** from the **Connect With** drop-down list and keep the default setting of the **Connection Type** as **Public**.

4. If you have not set a password yet, click **Generate Password** to generate a random password.

5. Copy the connection command and paste it into your terminal. The following is an example for macOS:

    ```bash
    mysql -u '<prefix>.root' -h '<host>' -P 4000 -D 'test' --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p'<password>'
    ```

</div>
<div label="TiDB Self-Managed">

After your TiDB Self-Managed cluster is started, execute your cluster connection command in the terminal.

> The following is an example connection command for MacOS:

Reviewer: ```suggestion
The following is an example connection command for macOS:
```

```bash

>  mysql --comments --host 127.0.0.1 --port 4000 -u root

Reviewer: ```suggestion
mysql --comments --host 127.0.0.1 --port 4000 -u root
```
